examples/standalone/index.js

```diff
@@ -1,10 +1,10 @@
-var i18n = require('i18n-future')();
+let i18n = require('i18n-future')();
```

Comment:
This example is broken because it's looking for the old package name here, we can fix that while we're here by updating the example's `package.json` to point to `hmpo-i18n` at `../..` and then updating this to `require('hmpo-i18n')();`

Reply:
Just wanted to make sure, this means updating the local package.json to the following:
`"dependencies": {
    "hmpo-i18n": "../.."
  }`

Reply:
Yeah exactly that